java/src/by/gsu/epamlab/task1/BusinessTrip.java

```diff
@@ -0,0 +1,73 @@
+package by.gsu.epamlab.task1;
+
+public class BusinessTrip {
+
+    private static final int DAILY_ALLOWANCE_RATE = 1000;
+    private String employeesAccount;
+    private int transportationExpenses;
+    private int numberOfDays;
+
+    public BusinessTrip(String employeesAccount, int transportationExpenses, int numberOfDays) {
+        this.employeesAccount = employeesAccount;
+        this.transportationExpenses = transportationExpenses;
+        this.numberOfDays = numberOfDays;
+    }
+
+    public BusinessTrip() {
+    }
+
+    public String getEmployeesAccount() {
+        return employeesAccount;
+    }
+
+    public void setEmployeesAccount(String employeesAccount) {
+        this.employeesAccount = employeesAccount;
+    }
+
+    public int getTransportationExpenses() {
+        return transportationExpenses;
+    }
+
+    public void setTransportationExpenses(int transportationExpenses) {
+        this.transportationExpenses = transportationExpenses;
+    }
+
+    public int getNumberOfDays() {
+        return numberOfDays;
+    }
+
+    public void setNumberOfDays(int numberOfDays) {
+        this.numberOfDays = numberOfDays;
+    }
+
+    public int getTotal() {
+        return ((transportationExpenses + DAILY_ALLOWANCE_RATE) * numberOfDays);
+    }
```

Comment:
No need for brackets due to priority of operations.

Reply:
You should take transportationExpenses only once. No need multiply it numberOfDay